components/Header/HeaderApp.js

```diff
@@ -8,10 +8,7 @@ import { style } from './HeaderAppStyle'
 export default class HeaderApp extends Component {
 
   render () {
-    let avatar = null
-    if (this.props.userId !== null) {
-      avatar = <AvatarApp userId={this.props.userId} />
-    }
+		const avatar = this.props.userId !== null && <AvatarApp userId={this.props.userId} />
```

Comment:
Unindent fix (my tab in VScode at job made this ... not my fault 👼 )
About `const avatar` it can't be false so I have to write this ternaire 2554718f746b05e9e074912e89eeb60f8963f537